pkg/registration/hub/csr/reconciler.go

```diff
@@ -129,24 +137,25 @@ func (b *csrBootstrapReconciler) Reconcile(ctx context.Context, csr csrInfo, app
 		return reconcileContinue, nil
 	}
 
-	err := b.accpetCluster(ctx, clusterName)
+	err := b.acceptCluster(ctx, clusterName)
```

Comment:
Mixing typos fixes with a real change make it harder to review and make it hard to maintain the code later. If you revert this fix because it caused a regression, we will lose also the typo fixes.

Reply:
good suggestion, I will drop the typos fixes in this pr